docs/installation/slack/cloud-slack.mdx

```diff
@@ -74,6 +74,22 @@ After this point, all of your commands will be executed on the default instance.
 
 ![Cloud Slack All Clusters](assets/cloud_slack_command_all_clusters.png "Cloud Slack all clusters")
 
+### Setting Public Alias for Private Channels
+
+In order to safeguard your confidentiality while enabling seamless integration with Botkube plugins, it's essential to establish a public alias for your private Slack channels. This alias will be accessible solely to Botkube Cloud administrators.
```

Comment:
BTW probably we should also mention that in some troubleshooting section? Something like:

- I can't see my Slack private channels in Cloud Dashboard

Reply:
Good idea Mati. I think that this could be also addressed to UX to display it somewhere next to channel table on UI
@miklosp let me know if you need more info about that.